src/index.css

```diff
@@ -75,20 +79,34 @@ a {
   background-color: whitesmoke;
   display: flex;
   flex-direction: column;
-  justify-content: center;
+  justify-content: flex-start;
   align-items: center;
+  padding-bottom: 2em;
 }
 
 .card {
   display: flex;
   background-color: white;
   height: 10em;
   width: 75%;
+  min-width: 55em;
   justify-content: space-around;
   margin-top: 2em;
+  padding-left: 1em;
   box-shadow: 0 5px 10px rgb(0, 0, 0, 0.1);
 }
 
+.card #removeBtn {
+  color: #4386bf;
+  border: none;
+  background-color: white;
+  padding-top: 0%;
+  padding-bottom: 0%;
+  margin-top: 1.1em;
+  margin-bottom: 0%;
```

Comment:
- [ ] Following CSS best Practices, it would be better to use shorthand properties.
  - [ ]  Instead of using padding-top and padding-bottom separately you can simply use just one line: `padding: 0% 0`.
  - [ ] Instead of using margin-top and margin-bottom separately you can simply use just one line: `margin: 1.1rem 0 0% 0`